tests/callbacks/test_early_stopping.py

```diff
@@ -228,7 +228,13 @@ def validation_epoch_end(self, outputs):
     early_stopping = EarlyStopping(
         monitor="abc", stopping_threshold=stopping_threshold, divergence_threshold=divergence_theshold
     )
-    trainer = Trainer(default_root_dir=tmpdir, callbacks=[early_stopping], overfit_batches=0.20, max_epochs=20)
+    trainer = Trainer(
+        default_root_dir=tmpdir,
+        callbacks=[early_stopping],
+        limit_train_batches=0.2,
+        limit_val_batches=0.2,
+        max_epochs=20,
+    )
```

Comment:
not related to this PR. I guess we need to update some of the tests here. they run for 20 epochs .. might improve ci runtime a little bit.

Reply:
Yes, the early stopping one is quite consuming. Definitely worth optimizing, same for the fault-tolerant ones.

Reply:
Im macos test env, the test case `test_fast_forward_sampler_with_distributed_sampler` cost much more time than that (about 60s) which I think might also affect ci runtime.